jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -98,16 +97,41 @@ public boolean isClosed()
 
     @Override
     public Connection acquire()
+    {
+        return acquire(true);
+    }
+
+    /**
+     * <p>Returns an idle connection, if available;
+     * if an idle connection is not available, and the given {@code create} parameter is {@code true},
+     * then schedules the opening of a new connection;
```

Comment:
Does it always create a new connection?    This should say `if possible` or `may schedule....` . At the very least refer to `tryCreate` for the actual strategy applied.